crates/transaction-pool/src/error.rs

```diff
@@ -232,6 +232,9 @@ pub enum InvalidPoolTransactionError {
     /// invocation.
     #[error("intrinsic gas too low")]
     IntrinsicGasTooLow,
+    /// Cross chain transaction didn't pass supervisor validation
+    #[error("cross chain transaction invalid")]
+    CrossTxInvalid,
```

Comment:
this error should be converted into a standalone error struct that then also implements `PoolTransactionError` this way we don't need this additional error variant and can use `Other`

Reply:
ye, this is an op specific error, so it doesn't make sense here

Reply:
done